src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, Color color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round(Math.PI * (radius * radius));
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: "
+                + this.getClass().getSimpleName().toLowerCase() + ", "
+                + "area: " + this.calculateArea() + " sq. units, "
+                + "radius: " + this.radius + ", "
+                + "color: " + getColor().toString()
+                + '}';
```

Comment:
The `toString` method should ensure consistent punctuation. Consider adding a space before the closing curly brace to maintain consistency in the output format.